test/games/strategy/engine/framework/mapDownload/DownloadFileParserTest.java

```diff
@@ -15,13 +15,13 @@
   @Test
```

Comment:
Format and apply refactor of "(new DownloadFileParser()).parse() -> DownloadFileParser.parse()"